apps/Learning-Buddy/config/colors.js

```diff
@@ -4,9 +4,9 @@ export const colors = {
   red: '#FF0000',
   blue: '#0D2FB8',
   green: '#3CC982',
-  lightBlue: '#00B0FC',
+  lightBlue: '#45D0E7',
```

Comment:
This probably requires some reconfiguring of the colors provider, but majority of blue here is #00b0fc per [the scheme](https://www.figma.com/file/OqP7SIsdRJ7pHJzo467MIY/24-7-lb?type=design&node-id=520-773&mode=design&t=yA6BVuQML3n5lD9t-4). If you want to add a new type of blue (buttonBlue?) with #45D0E7 that would be good.

Reply:
If you want to add an issue around updating the colors config file, that works!

Reply:
As required by the wireframes, I've added another blue colour to the button and returned to the original colour scheme.